lib/squake/client.rb

```diff
@@ -11,6 +11,7 @@ class Client
       T.any(
         T::Array[T::Hash[Symbol, T.untyped]],
         T::Hash[Symbol, T.untyped],
+        String,
```

Comment:
In which circumstances (that aren't exceptions) does the squake-api return a String value?

Reply:
leftover from testing 👍🏼